test/e2e/gateway_api_test.go

```
@@ -423,6 +428,286 @@ func testGatewayAPIRBAC(t *testing.T) {
 	}
 }
 
+func testGatewayAPIDNS(t *testing.T) {
+	domain := "gws." + dnsConfig.Spec.BaseDomain
+
+	gatewayClass, err := createGatewayClass(operatorcontroller.OpenShiftDefaultGatewayClassName, operatorcontroller.OpenShiftGatewayClassControllerName)
+	if err != nil {
```

I expect that you'll need to ignore an error saying that this already exists.  It's okay if it already exists.

---

[createGatewayClass](https://github.com/openshift/cluster-ingress-operator/blob/920e509eb6c3f94a1500834c49eadeffa5ef09e8/test/e2e/util_gatewayapi_test.go#L298) returns the gatewayClass in case it already exists. `IsAlreadyExists` error is ignored within that function already.